lib/ansible/cli/pull.py

```diff
@@ -250,16 +281,22 @@ def run(self):
         # RUN the Checkout command
         display.debug("running ansible with VCS module to checkout repo")
         display.vvvv('EXEC: %s' % cmd)
-        rc, b_out, b_err = run_cmd(cmd, live=True)
+        rc, b_out, b_err = safe_output_env(run_cmd)(cmd, live=True)
```

Comment:
I'm a bit late here, but wouldn't it have been more simple to add an `env` arg to `run_cmd` and then pass that into `subprocess.Popen` instead of manipulating the local environment?

Reply:
i thought we had it already, but that is only on the module_utils version, instead of modifying the function, modifying the local env and restoring seemed the simplest and least intrusive, specially if we are eventually going to deprecate pull. I'm fine redoing it the other way.